11ty/definitions/adhd.md

```diff
@@ -0,0 +1,30 @@
+---
+title: ADD/ADHD
+slug: adhd
+defined: true
+speech: noun
+reading:
+- text: "What Is ADHD? Meaning, Symptoms & Tests"
+  href: https://www.additudemag.com/what-is-adhd-symptoms-causes-treatments/
+- text: "No Time To Confront Racism In Neurodiversity"
+  href: https://blackgirllostkeys.com/adhd/no-time-to-confront-racism-in-neurodiversity/
+- text: DSM-5 ADHD Diagnosis
+  href: https://www.cdc.gov/ncbddd/adhd/diagnosis.html
+
+---
+
+ADHD is a neurological disorder that affects someone's ability to focus or execute tasks. It is commonly thought of as "the hyper kid in class", though it is much more complex and nuanced.
+
+## Common Usage
+
+People often say "I'm so ADD/ADHD" when they're referring to short periods of hyperactivity or inability to focus. It's used colloquially much like "I'm so OCD," and has many of the same negative ramifications for neurodivergent folks.
```

Comment:
I might suggest breaking this into 

## Appropriate Usage

[Medical context]

## Inappropriate Usage

(And put here what you have under Common Issues)

## Alt Words

Rather than saying "I was so ADD/ADHD" when meant without the medical diagnosis context, instead use:

- distracted
- foggy
- hyper
- unclear
- unfocused